src/services/doppler-plan-client.ts

```diff
@@ -42,4 +42,20 @@ export class DopplerPlanClient {
     );
     return result;
   }
+
+  async getPlans(): Promise<PlanModel[]> {
```

Comment:
```suggestion
  async getFeaturedPlansBySubscribers(): Promise<PlanModel[]> {
```

Reply:
As we talked yersterday we can change the focus of this function to make it simpler. With an ordered dictionary for subscribers plan for example.